strawberry/types/fields/resolver.py

```diff
@@ -94,27 +100,23 @@ class ReservedType(NamedTuple):
     """
 
     name: str
-    type: Type
+    type: Type[Any]
```

Comment:
For consistency I'll also type these as `type` instead of `Type[Any]`

Reply:
I wish it was the default :D